pages/osx/sed.md

```diff
@@ -29,3 +29,7 @@
 - Replace separator / by any other character not used in the find or replace patterns, e.g., #:
 
 `sed 's#{{find}}#{{replace}}#' {{filename}}`
+
+- Delete line number 123 from a file, in-place without saving a backup:
```

Comment:
Based on the example 3, and we can use this highlight `[]` to explain what `d` means:
```suggestion
- [d]elete the line at the specific line number in a file, overwriting the file:
```